scripts/examples/gn_efr32_example.sh

```diff
@@ -113,6 +113,8 @@ if [ "$#" == "0" ]; then
             enable Addition data advertissing and rotating device ID
         --use_ot_lib
             use the silabs openthread library
+        --use_chip_lwip_lib
+            use the chip lwip library
```

Comment:
what is the value of this flag compare to the regular ipv6 support ?
Which benefits i could obtain ?

Reply:
I'm not sure i understand the question... The goal of the flag is to be able to use the lwip version in the csa repository in case a feature is needed from lwip and it is not yet present in gsdk version used.

Reply:
Ok historically we were compiling GSDK's LWIP integration with this we can switch to the one from the Matter's SDK in case they bump the version faster than you do....